spec/services/attribute_asserter_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe AttributeAsserter do
+  include SamlAuthHelper
```

Comment:
Can we avoid including SamlAuthHelper here?  It seems like its not really being using (maybe I'm wrong).